### DISCUSSION TITLE
Automatic window size possible?

### DISCUSSION THREAD

**Discussion by woutable** [Eugeny/tabby#9951] [Q&A]
Tabby is only showing 26 lines, independent of the window size. Can I change this, so Tabby SSH-sessions are automatically using the maximum line numbers for its current window size?

**Comment by jnMetaCode** [comment]
Hi @benjamin605-creator! Looks like you are using the Chinese version — feel free to open an issue at [agency-agents-zh](https://github.com/jnMetaCode/agency-agents-zh/issues) for better support in Chinese.

Quick answer: after installing agents to OpenClaw, the `/agents` command depends on your OpenClaw + CrewClaw setup. Make sure your OpenClaw workspace has the agent files in the correct directory (`~/.openclaw/agency-agents/`). If the bot shows "None", it likely means the workspace path is not linked to your Telegram bot instance.

For detailed help in Chinese, please create an issue at the link above 👆
